manifests/plugin/certguard.pp

```diff
@@ -0,0 +1,4 @@
+class pulpcore::plugin::certguard {
```

Comment:
```suggestion
# @summary Pulp Certguard plugin
class pulpcore::plugin::certguard {
```

This is helpful to have because we generate docs automatically from these strings.